File: app/src/main/java/com/example/keyframework/adapter/DemoAdapter.java
package com.example.keyframework.adapter;
import androidx.annotation.Nullable;
import com.chad.library.adapter.base.BaseQuickAdapter;
import com.chad.library.adapter.base.BaseViewHolder;
import java.util.List;

/**
 * @author Administrator
 */
public class DemoAdapter extends BaseQuickAdapter {


    public DemoAdapter(int layoutResId, @Nullable List data) {
        super(layoutResId, data);
    }

    @Override
    protected void convert(BaseViewHolder helper, Object item) {

    }
 
}

File: app/src/main/java/com/example/keyframework/adapter/DemoAdapter.kt
package com.example.keyframework.adapter

import com.chad.library.adapter.base.BaseQuickAdapter
import com.chad.library.adapter.base.BaseViewHolder
import com.example.keyframework.R
import com.example.keyframework.bean.UserBean

class DemoAdapter(data: MutableList<UserBean>?) : BaseQuickAdapter<UserBean, BaseViewHolder>(R.layout.activity_main, data) {
    override fun convert(helper: BaseViewHolder?, item: UserBean?) {

    }
}